cpnose.py

```diff
@@ -139,7 +139,7 @@ def finalize(self, result):
             logging.root.warn("Failed to stop zmq boundary")
 
     def wantFile(self, filename):
```

Comment:
If we were to run the tests after installation, then setup.py wouldn't need to be excluded because it wouldn't be there. If CellProfiler is installed using pip, we should move the logic in cpnose into a nose plugin and specify the switch settings we need in setup.cfg.